tests/Integration/Behaviour/Features/Scenario/Product/update_customization_fields.feature

```diff
@@ -0,0 +1,23 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags update-customization-fields
+@reset-database-before-feature
+@update-customization-fields
+Feature: Update product customization fields in Back Office (BO)
+  As a BO user
+  I need to be able to update product customization fields in the BO
+
+  Scenario: I add customization to product
+    When I add product "product1" with following information:
+      | name       | en-US: nice customizable t-shirt  |
+      | is_virtual | false                             |
+    And product "product1" type should be standard
+    When I define following customization field as customizationField1:
```

Comment:
I'm not a fan of splitting the CustomizationField creation for the UpdateProduct command because it doesn't represent what really happens, and it leaves a lot of logic in the test class:
- first create an ObjectModel entity
- then use it to get the values that will be inserted in the product command

I know behat tests are not designed to represent technical implementation behind, but in this case it's really too far from the actual expected input

The update commands only has reference but no data to update the existing field, and you can't mix update with creations this way

Reply:
How should I provide multiple customizations then?

Reply:
I think we could use this notation:
```
    And I update product product1 with following customization fields:
        | customization_id | type | name              | required | by_module |
        |                  | text | en-US: front text | 1        | 0         |
        | customization1   | text | en-US: back text  | 0        | 1         |
```

And if I'm correct in the context you can then use `$table->getColumnsHash()` to get the multiple array